lib/atlas/scaler/graph_persistor.rb

```diff
@@ -0,0 +1,26 @@
+module Atlas
+  class Scaler::GraphPersistor
+    def initialize(dataset, path)
+      @dataset = dataset
+      @graph   = GraphBuilder.build
+      @path    = path
+    end
+
+    def persist_graph
```

Comment:
> Agreed, but my (general) rule is that if a class name is self-describing, and has only one useful method (persist_graph), you can opt for a generically-named class method

One caveat to this: if you assign your "processor" class (`Persistor`) to a variable, and then pass it into a different method or don't call `persist/save_to` straight away, a descriptive method name is preferable to a generic one since it makes it clearer what is happening. e.g. it isn't obvious what `myvar.call(path)` does, but it is slightly clearer what `myvar.save_to(path)` does.